.github/workflows/php-compatibility.yml

```diff
@@ -9,3 +9,5 @@ on:
 jobs:
     build:
         uses: impress-org/givewp-github-actions/.github/workflows/php-compatibility.yml@master
+        with:
+            test_versions: "[ '7.2', '8.0' ]"
```

Comment:
Let's remember to bump the shared workflow to 7.2 after 3.0 is released.

Reply:
How should we track that todo?